findingaids/templates/findingaids/finding_aids_page.html

```diff
@@ -3,189 +3,266 @@
+                    </ul>
+                    <div class="text-primary col-md-12">
+                        page {{ page_obj.number }} of {{ num_pages }}
+                        <div class="col-md-12">
+                            <!-- more padding -->
+                            <p/>
+                        </div>
+                    </div>
+                </center>
 
-      {% if topic and thistopiclist %}
+                <!-- case 3: user is looking at browses by title -->
+            {% elif view == 'title' %}
+                <center>
+                    <ul class="pagination justify-content-center">
+                        {% if browse == 'all' %}
+                            <li class="active">
+                                <a href="{{ root_url }}?view=title&browse=all">ALL</a>
+                            </li>
+                        {% else %}
+                            <li>
+                                <a href="{{ root_url }}?view=title&browse=all">ALL</a>
+                            </li>
+                        {% endif %}
+                        {% for b in browselinks %}
+                            {% if browse == b %}
+                                <li class="active">
+                                    <a href="{{ root_url }}?view=title&amp;browse={{ b|urlencode }}">{{ b }}</a>
+                                </li>
+                            {% else %}
+                                <li>
+                                    <a href="{{ root_url }}?view=title&amp;browse={{ b|urlencode }}">{{ b }}</a>
+                                </li>
+                            {% endif %}
+                        {% endfor %}
+                    </ul>
+                </center>
+                <div class="col-md-12">
+                    <!-- just padding -->
+                    <p/>
+                </div>
+                <!-- wait till user has clicked on a link before displaying browses -->
+                {% if browse != 'all' %}
+                    <table class="table table-striped scrc-list">
+                        <thead>
+                            <tr class="etable-header">
+                                <th colspan="1">
+                                    All Finding Aids beginning with "{{ browse }}"
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for b in browses %}
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <a target="_blank" href="http://www.lib.uchicago.edu/e/scrc/findingaids/view.php?eadid={{ b.0|urlencode }}">{{ b.1 }}</a>
+                                        </strong>
+                                    </td>
+                                </tr>
+                            {% endfor %}
```

Comment:
This could be written as `{% for b0, b1 in browses %}` as well. Also, I'm wondering if we could get the findingaids base url from site settings or ini. I won't block for that though. It's not directly related to the issue you're solving. I just wanted to point it out. But I see this url is repeating. If we're not going to make it configurable, maybe we should at least put it in a constant.

Reply:
Great suggestion; thanks.